ocs_ci/ocs/rados_utils.py

```diff
@@ -389,12 +390,20 @@ def get_pg_log_dups_count_via_cot(osd_deployments, pgid):
             f"ceph-objectstore-tool --data-path /var/lib/ceph/osd/ceph-"
             f"{osd_id} --op log --pgid {pgid}  > /var/log/ceph/pg_log_{pgid}.txt"
         )
-        # Install jquery in the pod and get the current pg log dups count
-        osd_pod.exec_sh_cmd_on_pod("yum install jq -y")
-        out = osd_pod.exec_sh_cmd_on_pod(
-            f"cat /var/log/ceph/pg_log_{pgid}.txt | jq  '(.pg_log_t.log|length),(.pg_log_t.dups|length)'"
+        logger.info(
+            "Copy current pg log dups file to local folder and parse dups number"
+        )
+        temp_file = tempfile.NamedTemporaryFile(
+            mode="w+", prefix=f"pg_log_{pgid}", delete=True
+        )
+        osd_pod.copy_from_pod_oc_exec(
+            target_path=temp_file.name, src_path=f"/var/log/ceph/pg_log_{pgid}.txt"
+        )
+        res = exec_cmd(
+            f"cat {temp_file.name} | jq  '(.pg_log_t.log|length),(.pg_log_t.dups|length)'",
```

Comment:
i see you deleted jq install step is jq binayr present in pod?

Reply:
as the log above states: "Copy current pg log dups file to local folder and parse dups number" 
It means we do not need to have 'jq' binary installed on pod, it will be text file will be parsed from the slave after we copy it